### ISSUE TITLE
Add accessibility support (inverted color scheme, font sizes)

### ISSUE THREAD

**Issue by sypherspace** [wireapp/wire-desktop#299]
A contact of mine is struggling to use the app due to a lack of accessibility features.

"I get about five half-length lines on my 27" screen. Is there any chance of getting the text in an easier format? Black on white is kind of like staring into the headlights of a car for me."

Please add support for accessibility-friendly color schemes, such as white text on a black or dark grey background, as well as increasing font sizes within the application.

**Comment by helenakos** [comment]
@sypherspace Thanks for the feedback. It's not on the near term roadmap but I am noting down your request.

**Comment by Strit** [comment]
Or atleast just a Dark Mode, like the android app has.

**Comment by manasb** [comment]
+1 for dark mode!

**Comment by karinavb** [comment]
Adding my vote for dark theme and the ability to change font sizes in the app. Shouldn't be too hard to implement and the existing Android dark theme in particular is, for me, a major advantage over other communication apps.

**Comment by fortran77** [comment]
For some of us, the desktop app's current color scheme of dark grey text on a light grey background is hard to read, but so is white text on a black background. Please also support plain black text rgb(0, 0, 0)  on a plain white background rgb(255, 255, 255).

**Comment by qazip** [comment]
Is there any news regarding a dark theme?

**Comment by helenakos** [comment]
@qazip Hi, no news or committed date at this moment. Sorry to hear that you dread to use Wire on desktop because of this. This is a resourceful task for us and although it's on our wish list we won't be working on it in the next couple of months.

**Comment by kwaaak** [comment]
Dark mode on the desktop +1

**Comment by lectrode** [comment]
I'd also like to request this feature. This is a great app, but the white interface is very harsh on the eyes.

**Comment by rnelling** [comment]
I too would like a dark theme for the PC desktop application.  Would prefer to use the app rather than in-browser.

**Comment by drgibbon** [comment]
Dark mode would be nice, but the user definitely needs to be able to control the font size. On my computer the font in Wire is just too small, which makes it very unpleasant to use. The user being able to read the text is a pretty rudimentary "feature", so I'm surprised that this issue was opened nearly a year ago without any attention since!

**Comment by dgutkowsky** [comment]
Hello, any news on that? White Wire is hurting eyes, especially in the evening and at night!
It's a health issue indeed!

**Comment by ehodzic** [comment]
Being able to customize application theme to one's liking has a very positive impact on impression of users. The competition allows one to change themes. Surely Wire can allow at the very least picking among a few pre-determined themes if not allow more powerful customization?

**Comment by Strit** [comment]
Even the new Skype got a light and dark theme to switch between. 🙂

**Comment by madLyfe** [comment]
a dark theme like skype has(i hate saying it) would be nice. not sure about that gradient/image as the background of the contacts/conversations list either. just some pretty, solid, dark colors for the theme. the white burns the retinas.

**Comment by JitseB** [comment]
Dark theme! 😍

**Comment by ruxandra-valcu** [comment]
A dark theme would be very useful - as it is, using Wire at night leads to a lot of eye-fatiguing glare.

**Comment by FHTheron** [comment]
Dark themes have suddenly become a deal-breaker for a large number of users of almost all software. While it may not be on the roadmap, definitely know that not having a dark mode leaves a bad first impression.

As an example, https://jira.atlassian.com/browse/STRIDE-2 is the second ticket for a new product and it has received very little attention. It becomes a stale signal of priorities not aligning with the end users, but rather with the sales teams of companies who end up paying for the product.

Very few chat apps have dark modes on desktop (Ryver does it very well), so having it in Wire would be a small (but very effective) way of pushing Wire to the front of water cooler discussions about chat alternatives.

**Comment by OuTopos** [comment]
+1 for a dark theme.
Thanks

**Comment by BadPackets** [comment]
Dark theme for desktop client, when?

**Comment by Antoine-Derouin** [comment]
In my opinion, a dark interface is the only feature Wire lacks.

**Comment by BadPackets** [comment]
Wire Support - KG (Wire Support)
Jun 4, 2:54 PM CEST

Thanks for writing in and for you request.

As of now, there are no plans to add dark them on our immediate roadmap. It is something we'd really like to add in the future, but for now,  what's stopping us is design and development resources.

I'll make sure that your request makes it directly to the ears of our master product manager, and if there's anything else we can help with, don't hesitate to ask.

All the best,
Kate
Wire Support

**Comment by BadPackets** [comment]
Yes, that appears to be disabled. I did a quick comparison and Slack's client does have that option.

**Comment by RRREEEEEEEE** [comment]
Why is dark theme and font changing still not added? It takes less than 2 minutes (literally) to do it client-side in the web browser. I doubt wire desktop is much different.

**Comment by manasb** [comment]
@nyggidy 

It is indeed quite sad. Not only this, but if you look at the commits - a huge portion of the commits are maintenance-related, often from bots. It doesn't seem like there is much of any actual development work going on in the desktop application and that is sad. This combined with the terrible call performance (vast portion of my calls do not function well for more than 5 minutes) makes me wonder why I made all the effort with getting a number of my family and friends to install and use Wire - they were happy with alternatives like WhatsApp etc. Should I try to use the web version of wire with a custom dark-style, I am faced with an hours-long login process which often doesn't complete before I shut my computer off.

**Comment by pyoung21** [comment]
For those still interested in a dark theme, I made a dark theme fork for me and my friend feel free to check it out, you can basically inject any custom css into the electron webview, I guess that's all we can do right now.

**Comment by Jfamig** [comment]
Hi @pyoung21 , I've been trying to find a way to contact you, but I have recently installed wire and would like some help installing your darkmode fork. How do you install it, exactly?

**Comment by pyoung21** [comment]
@Jfamig Which platform are you targeting? You can build it yourself by following the instructions in the README file. I'll make a release later so you can downlad it directly if you're unable to build it yourself.

**Comment by Jfamig** [comment]
@pyoung21 Win10 64bit

I have little experience with git and I tried following the instructions but I had a hard time getting the repo to build. 

I would really appreciate a direct download so I can share it with my friends who use wire as well, who also have little experience building from github. Thanks! :D

**Comment by pyoung21** [comment]
Windows build available on the project's repo (https://github.com/pyoung21/DarkWire/releases).

**Comment by Jfamig** [comment]
@pyoung21 got this upon clicking on the launcher. https://i.imgur.com/VjC0tiW.png

Log: https://pastebin.com/AWcjgX1u

I probably messed something very basic up, but I tried my own troubleshooting like closing wire and restarting. Still no luck.

**Comment by hoodieak** [comment]
PLEASE DARK MODE FOR WINDOWS thanks in advance, cool to see theres a github for this

**Comment by maximbaz** [comment]
If you are feeling adventurous and willing to compile desktop app from sources, it is relatively easy to add Dark Theme. I did this for myself and described the steps here: [HOWTO](https://gist.github.com/maximbaz/e80a105f4f16813f4b2549e70787e984).

The theme itself is taken from https://github.com/wireapp/wire-webapp/pull/3434. It looks awesome right now, but there is no guarantee it will not break one day as the UI will change.

Enjoy 😉 

![screenshot](https://user-images.githubusercontent.com/17581244/40428739-44a9e86c-5ea1-11e8-90e8-ace4f62d1bc3.png)

**Comment by manasb** [comment]
@maximbaz Thank you

**Comment by Termuellinator** [comment]
It really is a shame that there is a readymade pullrequest but it won't get merged due to support-reasons. To be brutally honest: the support-requirements for what is basically a theme are not really astronomical imho :/
Given that the wish for a dark mode is voiced for years now...

**Comment by tech189** [comment]
Another vote here for a dark theme! And if it's possible, it would be nice to link it to the macOS system wide dark mode so that the theme switches in Wire when the system is in dark mode.

**Comment by ffflorian** [comment]
By the way, this was implemented in https://github.com/wireapp/wire-webapp/pull/5415 and will be published soon :tada:

**Comment by manasb** [comment]
Thank you!
